fixtures/locations.yaml

```diff
@@ -0,0 +1,8 @@
+locations:
+    -
+        name: Austin Recycle and Reuse Drop-off Center
```

Comment:
I've been passing the contact name in -- 
we'll need to update the ContactDetails module to not pass in name to the ContactAddress module
and the ContactAddress module to use location.name
src/js/modules/ContactDetails and src/js/modules/ContactAddress